tools/cardano-sl-tools.cabal

```diff
@@ -306,9 +306,6 @@ executable cardano-launcher
                        BangPatterns
                        ScopedTypeVariables
 
-  if os(windows)
```

Comment:
Can you justify this change? The commit message doesn't have any clarifications :/